include/eve/module/core/regular/impl/abs.hpp

```diff
@@ -27,20 +27,22 @@ namespace eve::detail
   {
     if constexpr(O::contains(saturated2))
     {
-      if      constexpr( signed_integral_scalar_value<T>  ) return  ( (v == valmin(eve::as(v)))
-                                                                    ? valmax(eve::as(v)) : eve::abs(v)
-                                                                    );
-      else if constexpr( signed_integral_simd_value<T>    ) return if_else( v == valmin(eve::as(v))
-                                                                          , valmax(eve::as(v)), eve::abs(v)
-                                                                          );
-      else                                                  return eve::abs(v);
+      if constexpr( signed_integral_scalar_value<T>  ){
+        if(v == valmin(eve::as(v))) return valmax(eve::as(v));
```

Comment:
Mask should be good too as we saw earlier ?

Reply:
non j'ai essayé